### DISCUSSION TITLE
LPC11U24 project support (bare metal profile)

### DISCUSSION THREAD

**Discussion by webgal** [mbed-ce/mbed-os#338] [Q&A] [answered]
Does anyone know if mbed-ce can compile mbed projects based on the LPC11U24? It doesn't appear in the list of supported targets, and we have several projects deployed using mbed's compiler. I'm concerned that once ARM discontinues their online compiler, we won't be able to provide any bug fixes or maintenance.

I don't yet have the skills to add a new target, but I would be willing to donate beer or even money if someone could help.

PS: The announcement from mbed is bittersweet. It saddens me, but I'm also glad that the community has taken over.

**Comment by JohnK1987** [comment]
Hello,

your target seems to be supported in Mbed2 and maybe MbedOS 5 (not sure if this was ever working) then the ARM Mbed left support of this target with move from MbedOS 5 to MbedOS6.
MbedCE started from MbedOS6 so also MbedCE do not have source files for your target.

I tried to copy these old data from MbedOS5 but i am not able to test it. So if you have a board for a simple test of blinky led with serial output...

https://github.com/JohnK1987/mbed-os/tree/NXP_LPC11U24

BR, Jan

**Reply by JohnK1987** [reply]
@webgal did you check?

**Reply by JohnK1987** [reply]
@webgal ???

**Reply by JohnK1987** [reply, marked as answer]
Ok, this was probably just waste of my time. This will be closed and source files deleted.
